cgi-bin/Apache/BML.pm

```diff
@@ -1916,7 +1916,7 @@ sub get_path_info
 sub get_remote_ip
 {
     my $apache_r = BML::get_request();
```

Comment:
Any objections to making these functions in `Apache::BML` be a lightweight wrapper to `DW::Request`?

BML is probably, sadly, not going to die anytime soon.